platform.md

```diff
@@ -832,6 +832,12 @@ The following variables SHOULD be set in the lifecycle execution environment and
 
 The platform SHOULD NOT assume any other stack-provided environment variables are inherited by the buildpack.
 
+##### Builder-Provided Variables
+The following variables MAY be set in the lifecycle execution environment and SHALL be directly inherited by the buildpack without modification:
```

Comment:
```suggestion
The following variables MAY be set in the lifecycle execution environment and SHALL be directly inherited by the buildpack without modification, if the buildpack implements a buildpack API that supports asset packages:
```

Reply:
@natalieparellano I think you are right that we need a caveat here but maybe we want something more generic? It seems weird to apply this specific carve-out to an entire table that could be extended to include more variables.

Reply:
Makes sense! I modified my suggestion.